RELEASE.md

```diff
@@ -202,6 +202,10 @@
 
 ## Versione 11.12.5 (30/05/2024)
 
+### Novità
+
+- Per ragioni di conformità alle linee guida Agid, inibito l'inserimento di CT File nella cartella modulistica.
```

Comment:
in io-comune v2 giusto?

Reply:
v3

Reply:
allora perchè è una pr per il branch 11.x.x?
andrebbe fatta per il main

Reply:
Il branch 11.x.x è per tutti i comuni che non hanno ancora fatto la migrazione a Slate (tutti praticamente)
main è per quelli che hanno già slate
per v2 il branch è v2 :)

Reply:
ok, quindi immaginando che questa sia una modifica fatta solo sul BE, dovrebbe esserci anche una pr analoga per il main, oltre che per la 11.. o ho capito male?

Reply:
https://github.com/RedTurtle/design-comuni-plone-theme/pull/790
sìsì, eccola qui